src/config-hardware/audio-apps.md

```diff
@@ -0,0 +1,39 @@
+# Audio Applications & Extensions
+
+These applications provide control and customization settings for audio devices beyond the `Settings` menu.
+
+---
+
+## EasyEffects
+
+[EasyEffects](https://github.com/wwmm/easyeffects) is a third-party application that can apply audio effects and filters to audio input and output streams. You can download and install EasyEffects from the Pop!\_Shop.
```

Comment:
This doesn't explain what "effects and filters" are/which ones there are or how to use the application.

Reply:
It was suggested that EasyEffects be mentioned. I can also include a mini-tutorial.

Reply:
Added a basic overview of the UI and how to add an effect.